charts/tfy-karpenter-config/templates/controlplane/karpenter-control-plane-nodepool.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.karpenter.controlPlaneNodePool.enabled }}
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: control-plane-nodepool
```

Comment:
Do we want to keep the name fixed or have it from values.yaml. It will be better if we can take it from values.yaml
`{{ .Values.karpenter.controlPlaneNodePool.name }}`